jme3-core/src/com/jme3/gde/core/sceneexplorer/Bundle.properties

```diff
@@ -1,4 +1,4 @@
 CTL_SceneExplorerAction=SceneExplorer
 CTL_SceneExplorerTopComponent=SceneExplorer
 HINT_SceneExplorerTopComponent=The SceneExplorer provides an Overview over the SceneGraph of your Scene.
-SceneExplorerTopComponent.jButton1.text=update
+SceneExplorerTopComponent.jButton1.text=Update
```

Comment:
Please do not change the button label. Also the label of the **'attach...'** button in the `AppStateExplorer` tab is all lowercase.